Update configured WL branches

rossjrw opened the pull request:
The WL config in this repository was out of date. I found a more recent one which is currently being used by WL EN: http://topia.wikidot.com/wl-interwiki-inc

This PR updates our config to match that. Note that I have removed the "title" property from each of the configs, as they appear to be hardcoded to English, so they'll only be relevant to a single branch and therefore won't be very useful.

Should only be merged after #2.

rossjrw committed:
Update configured WL branches

emmiegit commented on js/branches-info-wl.js:
```diff
@@ -1,37 +1,100 @@
     name: "English",
     head: "Languages",
-    url: "https://wanderers-library.wikidot.com/",
+    url: "http://wanderers-library.wikidot.com/",
```
Wikidot supports these URLs, redirecting them to the HTTP version. I've generally preferred the HTTPS links because if/when it switches then it'll make the transition easier if some links are already updated.

rossjrw replied:
Sounds good, will HTTPS-ify all of them shortly.

Worth noting that that's not always the case - the ajax module connector endpoint for example will reject a HTTPS connection if the site is set up only to use HTTP (and vice versa, if the site is set up *only* to use HTTPS) - but the only module in use for the Interwiki currently is `quickmodule.php` which doesn't seem to care.

emmiegit replied:
Oh jeez, is that the case? Wikidot why

rossjrw committed:
Switch all WL URLs to HTTPS

Wikidot usually freely interconverts HTTP/HTTPS connections, so HTTPS is
a sensible default.

Note that this should be reevaluated if more endpoints are added, as
some Wikidot endpoints are particular about how secure their connection
is. This is the case for ajax-module-connector.php, for example, but is
not the case for quickmodule.php, which is currently used.

rossjrw commented:
Moving this back to draft - not all of the information in the 'updated' config is accurate

rossjrw committed:
Remove nonexistent WL branches
